lib/platform/gitea/index.ts

```diff
@@ -587,6 +587,16 @@ const platform: Platform = {
     return config.issueList;
   },
 
+  async getIssue(number: number, useCache = true): Promise<Issue> {
+    // istanbul ignore if
+    if (!useCache) {
+      delete config.issueList;
+    }
+    const issueList = await platform.getIssueList();
```

Comment:
This doesn't work as expected with `useCache=false` as `getIssueList` would use the cached http request. Sending a commit to fix it 🙃